frontends/web/src/components/auth/authrequired.tsx

```diff
@@ -89,18 +89,21 @@ export const AuthRequired = () => {
         textCenter
         verticallyCentered
         withBottomBar>
-        <ViewHeader small title={t('auth.title')} />
-        <ViewContent children={undefined} minHeight="0" />
-        <ViewButtons>
-          <Button
-            autoFocus
-            primary
-            hidden={authForced.current}
-            disabled={authenticating}
-            onClick={newAuthentication}>
-            {t('auth.authButton')}
-          </Button>
-        </ViewButtons>
+        { !authenticating && (
```

Comment:
a bit hard to undestand to me - `authenticating` is false by default, so how does this help in not showing the auth stuff at startup?

why this and not `authRequired && ( ... )`?